pkg/utils/image/infos.go

```diff
@@ -77,13 +83,17 @@ func GetImageInfo(image string, cfg config.Configuration) (*ImageInfo, error) {
 		registry = ""
 	}
 
-	return &ImageInfo{
+	imageInfo := &ImageInfo{
 		Registry: registry,
 		Name:     name,
 		Path:     path,
 		Tag:      tag,
 		Digest:   digest,
-	}, nil
+	}
+
+	imageInfo.Reference = imageInfo.String()
```

Comment:
Why do it it like this? Could all be done in the creation of the pointer IMO. Maybe I am missing something.